### PR TITLE
add HStackInvertedLists example to n indices example

### PR DESCRIPTION
**Pull request by rom1504**

### CODE REVIEW

File: examples/distributed_autofaiss_n_indices.py [hunk 1]
```diff
@@ -78,3 +78,52 @@
...
+        ilv = faiss.InvertedListsPtrVector()
+
+        for fname in invlist_fnames:
+            if os.path.exists(fname):
+                index = faiss.read_index(fname, faiss.IO_FLAG_MMAP)
+                index_ivf = faiss.extract_index_ivf(index)
+                il = index_ivf.invlists
+                index_ivf.own_invlists = False
+            else:
+                raise AssertionError
```
Comment (hitchhicker) [inline examples/distributed_autofaiss_n_indices.py:111]: Should it raise FileNotFoundError exception instead ?

File: examples/distributed_autofaiss_n_indices.py [hunk 2]
```diff
@@ -78,3 +78,52 @@
...
+            ilv.push_back(il)
+
+        self.big_il = faiss.HStackInvertedLists(ilv.size(), ilv.data())
+        ntotal = self.big_il.compute_ntotal()
+
+        self.index = faiss.read_index(invlist_fnames[0], faiss.IO_FLAG_MMAP)
+
+        index_ivf = faiss.extract_index_ivf(self.index)
+        index_ivf.replace_invlists(self.big_il, True)
+        index_ivf.ntotal = self.index.ntotal = ntotal
```
Comment (hitchhicker) [inline examples/distributed_autofaiss_n_indices.py:121]: Where will we use the `index_ivf` ?

**Review by hitchhicker — COMMENTED** [review]
Thanks for the PR !
Left one comment and one question.

File: examples/distributed_autofaiss_n_indices.py (hunk 1 above)
Comment (rom1504) [reply examples/distributed_autofaiss_n_indices.py:111]: Yeah why not, will fix

File: examples/distributed_autofaiss_n_indices.py (hunk 2 above)
Comment (rom1504) [reply examples/distributed_autofaiss_n_indices.py:121]: Index ivf is part of index, so when you use index you use index ivf
There are pointers

**Pull request merged**